ols/app/models/config.py

```diff
@@ -268,7 +303,14 @@ def __eq__(self, other):
 
     def validate_yaml(self) -> None:
         """Validate conversation cache config."""
-        pass
+        if self.type is not None:
+            match self.type:
+                case constants.REDIS_CACHE:
+                    return self.redis.validate_yaml()
+                case constants.IN_MEMORY_CACHE:
+                    return self.memory.validate_yaml()
+                case _:
+                    raise InvalidConfigurationError("unknown conversation cache store")
```

Comment:
I'd specify the type in exception message, so the user will know that he (for example) made a typo...